package.json

```diff
@@ -58,6 +58,7 @@
     "lint": "npm run lint-core",
     "lint-fix": "npm run lint-core -- --fix && npm run lint-addons -- --fix && npm run lint-examples -- --fix && npm run lint-docs -- --fix && npm run lint-editor -- --fix && npm run lint-playground -- --fix && npm run lint-manual -- --fix && npm run lint-test -- --fix && npm run lint-utils -- --fix",
     "test-unit": "qunit -r failonlyreporter -f !-webonly test/unit/three.source.unit.js",
+    "test-unit-addons": "qunit -r failonlyreporter -f !-webonly test/unit/three.addons.unit.js",
```

Comment:
@mrdoob Are you fine with reactivating unit tests for addons?

Reply:
I'm personally in favor of having unit tests for addons. In several cases I've found it more more difficult to test changes (GLTFExporter and BufferGeometryUtils for example) without them, and would be easier with unit tests.

That said ... deleting the addons unit tests back in #23352 was a pretty big change! I feel like we should at least open an issue to address why we're adding them back, and what (if anything?) should be different this time. Without that I worry that new unit tests could be deleted again, and this isn't an encouraging outcome for contributors.

Reply:
I created this one: https://github.com/mrdoob/three.js/issues/28452

Reply:
At the end of the day it's all about how much value they bring versus how much maintenance and discussions they cost.